ddl/column.go

```diff
@@ -1075,7 +1075,13 @@ func (w *worker) doModifyColumnTypeWithData(
 				w.reorgCtx.cleanNotifyReorgCancel()
 				return ver, errors.Trace(err)
 			}
-			if err1 := t.RemoveDDLReorgHandle(job, reorgInfo.elements); err1 != nil {
+			var err1 error
+			if AllowConcurrentDDL.Load() {
+				err1 = w.RemoveDDLReorgHandle(job, reorgInfo.elements)
+			} else {
+				err1 = t.RemoveDDLReorgHandle(job, reorgInfo.elements)
+			}
```

Comment:
there are many patterns like this, we can make a function like `RemoveDDLReorgHandle`, then the caller just call this function, no need consider the switch